rs/tests/os_qualification/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+load("//rs/tests:common.bzl", "COUNTER_CANISTER_RUNTIME_DEPS", "DEPENDENCIES", "GUESTOS_RUNTIME_DEPS", "MACRO_DEPENDENCIES", "MAINNET_REVISION_RUNTIME_DEPS", "UNIVERSAL_VM_RUNTIME_DEPS", "XNET_TEST_CANISTER_RUNTIME_DEPS", "MAINNET_NNS_CANISTER_RUNTIME_DEPS", "MAINNET_NNS_CANISTER_ENV")
+load("//rs/tests:system_tests.bzl", "system_test")
+load("@rules_rust//rust:defs.bzl", "rust_library")
+
+package(default_visibility = ["//rs:system-tests-pkg"])
+
+system_test(
+    name = "guest_os_qualification",
+    env = {
+        "XNET_TEST_CANISTER_WASM_PATH": "$(rootpath //rs/rust_canisters/xnet_test:xnet-test-canister)",
+    } | MAINNET_NNS_CANISTER_ENV,
+    proc_macro_deps = MACRO_DEPENDENCIES,
+    tags = [
+        "manual",
+    ],
+    target_compatible_with = ["@platforms//os:linux"],  # requires libssh that does not build on Mac OS
+    uses_guestos_dev_test = True,
+    runtime_deps =
+        GUESTOS_RUNTIME_DEPS +
+        UNIVERSAL_VM_RUNTIME_DEPS +
+        MAINNET_REVISION_RUNTIME_DEPS +
+        MAINNET_NNS_CANISTER_RUNTIME_DEPS +
+        COUNTER_CANISTER_RUNTIME_DEPS +
+        XNET_TEST_CANISTER_RUNTIME_DEPS,
+    deps = DEPENDENCIES + ["//rs/tests", ":os_qualification_utils"],
```

Comment:
Could you specify dependencies explicitly instead of using the huge `DEPENDENCIES` which will include a lot of dependencies you don't need.

Reply:
Also you can drop `//rs/tests` here.